small-talk/app/accountCreate/page.js

```diff
@@ -42,6 +44,32 @@ export default function RegisterPage() {
           return;
         }
         
+        if(!(await validateInput(username))){
+          alert("Username: Too many characters or Use of prohibited characters");
+          return;
+        }
+
+        if(await vulgar(username)){
```

Comment:
I was wondering if we should also check the password for vulgar language as well?

Reply:
Sure we could. I initially left it out since making vulgar passwords wont really offend anyone since only the user should know it

Reply:
Just applied it. Thank you